release-please-config-yoshi-submodules.json

```diff
@@ -391,4 +388,4 @@
     "plugins": [
         "sentence-case"
     ]
-}
+}
```

Comment:
```suggestion
}

```

Reply:
gah, I need to turn off this format on autosave I guess!